### PR TITLE
Dialog: Stop tracking instance in destroy() to avoid memory leaks

### PR DESCRIPTION
**Pull request by tjvantoll**
Fixes #11125

### CODE REVIEW

File: ui/dialog.js [hunk 1]
```diff
@@ -155,6 +155,7 @@ return $.widget( "ui.dialog", {
 		var next,
 			originalPosition = this.originalPosition;
 
+		this._stopTrackingInstance();
```
Comment (jzaefferer) [inline ui/dialog.js]: Can't you just call `._untrackInstance()`?

Comment (tjvantoll) [inline ui/dialog.js]: Yeah I like that name better.

Comment (scottgonzalez) [inline ui/dialog.js]: He didn't mean rename it, he meant there's already a function that does this, and it's called `_untrackInstance()`.

Comment (tjvantoll) [inline ui/dialog.js]: doh

Comment (tjvantoll) [inline ui/dialog.js]: Yeah that method works and it's even written nicer than mine. Changed.

**Comment by scottgonzalez** [comment]
I approve of this one line change ;-)

**Comment by jzaefferer** [comment]
:+1:

**Pull request closed without merging**